internal/image/resource_image_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/dustinkirkland/golang-petname"
+	petname "github.com/dustinkirkland/golang-petname"
```

Comment:
We've usually been avoiding this kind of no-change import renaming in both Incus and the Terraform provider.